application/SFTP.py

```diff
@@ -112,11 +112,13 @@ def rename(self, cwd, old, new):
 
         return True, ''
 
-    def remode(self, cwd, new_mode):
```

Comment:
might better call it "chmod" since we are **ch**anging the mode anyways. is there any reason we called it 'remode'?